src/Splitio/Services/Client/Interfaces/ISplitClient.cs

```diff
@@ -96,6 +96,24 @@ public interface ISplitClient : ISplitClientAsync
         /// </returns>
         Dictionary<string, SplitResult> GetTreatmentsWithConfig(Key key, List<string> features, Dictionary<string, object> attributes = null);
 
+        /// <summary>
+        /// Same as GetTreatmentsWithConfig but this method evaluate by FlagSets and returns a Dictionary<string, string> containing the resulting treatment for each feature flag evaluated.
```

Comment:
Should it be `Dictionary<string, SplitResult>` ? since it returns the treatment with config object.